candy-machine/program/src/processor/mint.rs

```diff
@@ -571,6 +579,7 @@ pub fn handle_mint_nft<'info>(
         ctx.accounts.rent.to_account_info(),
         candy_machine_creator.to_account_info(),
     ];
+
```

Comment:
Hm we'd have to do some weird logic way earlier in the transaction to catch this I think. I dont believe its safe to bot tax here. As of now this should still be safe since the freeze should fail later on, but good point yeah.

Reply:
Another thing though. Not actually sure it makes sense to check this. You would have to be maliciously sending in instructions to make this the case. I'm under the impression that bot tax should be used to prevent bots trying to mint from killing the network, and not to prevent a malicious actor from just bringing it down.

Reply:
If youre just asking if freeze is set in general, the freeze ix from token metadata should just fail later on if it doesnt have freeze authority.